src/modules/app/__tests__/fetchData.spec.js

```diff
@@ -72,7 +72,7 @@ describe('fetchData', () => {
   it('should fail if json is malformed', async () => {
     const malformedJSON = 'I\'m so mean!'
     const dispatch = jest.fn()
-    const oldPayload = new Payload(false, 'https://weird-endpoint/old-url/api.json', {}, null)
+    const oldPayload = new Payload<CityModel[]>(false, 'https://weird-endpoint/old-url/api.json', [], null)
```

Comment:
```suggestion
    const oldPayload = new Payload<Array<CityModel>>(false, 'https://weird-endpoint/old-url/api.json', [], null)
```

Reply:
As we have no convention on that, I'd just leave it like this for now....